js/songs/index.html

```diff
@@ -0,0 +1 @@
+<h2>I'm the songs index</h2>
```

Comment:
Say you did have `this.songs` defined in your controller. How would you go about displaying **each** items in `this.songs` in your view?